CONTRIBUTING.md

```diff
@@ -83,7 +83,12 @@ For the `cc-global-header` component, you will need to add Axios via CDN:
 ```html
 <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
 ```
-However, if for some reason the API call is unsuccesful, maybe due to Wordpress being unavaible, there are dummy Menu items that will display but this feature is available only for Development environments. 
+
+In your development setup, we have a `isDevelopmentMode` attribute you can pass to the CC Global Header element to render placeholder Menu items.
+
+```html
+<cc-global-header donation-url="http://example.com" isDevelopmentMode />
```

Comment:
Let's use the kebab case since HTML is not case-sensitive.
```suggestion
<cc-global-header donation-url="http://example.com" is-development-mode />
```